data_store/utils.py

```diff
@@ -0,0 +1,78 @@
+
+from enum import IntEnum
+import fnmatch
+import os
+from pathlib import Path
+from typing import List
+
+
+class DataTypes(IntEnum):
+    """Constants for data types."""
+    DATASET = 0
+    CHARSET = 1
+    PALETTE = 2
+    DICOMDIR = 3
+
+
+class DataStore:
+    """Interface to pydicom-data data storage.
+
+    Attributes
+    ----------
+    data_path : pathlib.Path
+        The absolute path to the data directory.
+    """
+    def __init__(self) -> None:
+        """Initialise a new Interface."""
+        lib_dir = os.path.dirname(os.path.realpath(__file__))
+        self.data_path = Path(lib_dir, '../', 'data').resolve()
+
```

Comment:
Can get lib_dir similar to other comment with `Path(__file__).resolve().parent` or even straight to `self.data_path = Path(__file__).resolve().parent.parent / "data"`

Reply:
Done